### ISSUE TITLE
Province Abandonment

### ISSUE THREAD

**Issue by Firesoul7** [PhelanBavaria/ancienttimeline#7]
I propose that at a later time a system be created in order to simulate province abandonment. Many provinces were occupied at one time then later deserted. I already have some provinces lose their owners through use of unused tags (but this solution would not be used for the dynamic system). What I suggest however is a dynamic system of events and such which would take place in all countries, and in essence put a size limit on Empires. The frequency of events would depend on the type of terrain of each province and other factors such as stability etc, and would be tied to provinces and not countries. In each event the owner of the province receiving the event would be given a choice between a loss of prestige, money, legitimacy, stability in some cases, etc, or a loss of development in the province, resulting ultimately in the province becoming abandoned.

**Comment by qweytr** [comment]
I agree that some way to make colonized provinces uncolonized could be useful.

**Comment by PhelanBavaria** [comment]
I like that idea too, but I would like to see a specific example in history for when that happens, so we have a better idea of what to do.

**Comment by Firesoul7** [comment]
Well, a lot of cities were abandoned over time, and I can give lots of examples of that. Babylon, Ur, Petra, etc. Cities would slowly decline in population until everyone was gone. Province abandonment would be this on a larger scale. As for a historical example of full scale abandonment of an entire province, you could probably look into Egyptian history. The Nile would change course, or some Pharaoh would call for the abandonment of an area, or there would be an invasion killing lots of people, and an entire province would end up abandoned. You could in particular look into https://en.wikipedia.org/wiki/Akhenaten and the city of Amarna, which was built quite far from other Egyptian centres. Upon his death, the city was slowly abandoned, and the province along with it. 

This is how I would want the system to work. Well built up provinces with lots of cities would be unlikely to be abandoned, but a ruler could decide to establish a colony, and they could make it successful but maybe only if they sacrificed heavily. While Akhenaten kept his city strong, his entire empire fell to pieces. He was able to establish a colony and keep it going but only by working extremely hard on doing so. This is how I want colonisation to be. Possible most anywhere but balanced.

**Comment by Firesoul7** [comment]
I had an idea last night: there could be created a new building called 'Population expansion' or some such. Once built, this building would take away some manpower and money, remove itself and upgrade a province modifier from 'One city' to 'Two cities' to 'Three cities' etc, up to 'Ten cities'. After 10 cities if the building is built it would increase base tax, production and manpower in the province. Random events including disasters would lower the number of cities in the modifier, and if the province hits 'Zero cities' then it would decolonise by event.

**Comment by qweytr** [comment]
I think development models that well enough already.

**Comment by Firesoul7** [comment]
Unfortunately I don't believe province abandonment would work using the
development system.

**Comment by qweytr** [comment]
The development system is perfect for the province abandonment. Bad events and disasters could decrease the development of a province and if it hits zero, the province gets abandoned.

**Comment by Firesoul7** [comment]
Are you sure it allows that?

**Comment by qweytr** [comment]
Of course.

**Comment by PhelanBavaria** [comment]
Well, rather than using the province development, I would use the city_size as indicator for when it should be abandoned. Surely the province development should go down as well though.

**Comment by qweytr** [comment]
The city_size is a discontinued relic from EUIII. It has no function in EUIV. Development is the current equivalent.

**Comment by PhelanBavaria** [comment]
It does have a function. If you put city_size into a province file which is uncolonized, then it will show up as having a colonization progress.
Also, we can increase and decrease city_size, can't we?
Thirdly, are you sure that you can put development at 0?
As far as I remember you couldn't do that with the old base tax system.

**Comment by Firesoul7** [comment]
I recommend we create a test event, say have the trigger as tag = MCD, and have it rapidly lower base tax, production and manpower to zero in a certain province. In this way we'll be able to confirm with certainty that it would work. 

Sent from my Samsung device

-------- Original message --------

**Comment by qweytr** [comment]
city_size only has a function for colonies, not provinces with a city. It's not possible to put development to 0. What I meant was that the events that decrease development could make the province abandoned if the development is already 1.

**Comment by PhelanBavaria** [comment]
That could work, although we would have to give every province a development level according to it's population, which means that a freshly colonized province can't be 4 4 4 or something.

**Comment by qweytr** [comment]
The development should be mostly based on population anyway.
